include/deal.II/grid/reference_cell.h

```diff
@@ -1389,6 +1396,14 @@ ReferenceCell::face_reference_cell(const unsigned int face_no) const
 
 
 
+inline constexpr unsigned char
+ReferenceCell::default_combined_face_orientation()
+{
+  return 1u;
```

Comment:
If you felt compelled in a follow-up patch to explain why this value, I would accept such a pull request. If you wanted to do that, you could also write this as `0b010` to make it even clearer that you're treating this as a three-bit bitfield.